cmd/ll2go/main.go

```diff
@@ -218,8 +218,8 @@ func ll2go(llPath string, funcNames map[string]bool) (*ast.File, error) {
 			underlying = "int32"
 		case intSize < 64:
 			underlying = "int64"
-		default:
-			return nil, errors.Errorf("support for integer type with bit size %d not yet implemented", intSize)
+		default: // math.Big
+			underlying =  "Int"
```

Comment:
It's practical for pseudocode, but I can see that it would be a pretty significant effort to make it actually work. Google seems adamant about not taking on int128 as a native type, so this might be the only way to recover it.

If the goal is correct code, yeah panic makes more sense for now.